apps/package.json

```diff
@@ -117,7 +117,7 @@
     "messageformat": "0.2.2",
     "mkdirp": "0.5.1",
     "mochify": "2.18.0",
-    "moment": "2.10.6",
+    "moment": "^2.14.1",
```

Comment:
I would tie to a specific version (i.e. "2.14.1" instead of "^2.14.1") so as to avoid future confusion when engineers end up on slightly different versions.